HIP/hip-100.md

```diff
@@ -0,0 +1,479 @@
+---
+hip: 100
+title: Signature and Wallet Providers
+author: Daniel Akhteorv <daniel@launchbadge.com>
+type: Standards Track
+category: Application
+needs-council-approval: Yes
+status: Draft
+created: 2022-01-20
+discussions-to: TBD
+updated: 2022-01-20
+---
+
+## **Abstract**
+
+Ethereum like signature and wallet providers with the Hedera SDKs
```

Comment:
All Hedera SDKs or just the JS sdk?  Go and Java don't really map to some JS concepts well, would those be in another HIP?

Reply:
Primary requirement is for the JS SDK to implement this, although the same principles should be extended to all SDKs eventually.

Reply:
Added a simple message about the primary concern being the JS SDK.